src/components/ComboBox.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const ComboBox = props => {
+   const {options, label} = props;
+  return (
+    <div>      
+      <label className='combo_label'>{label}</label>
+      <select name="options" id="options" className='combo_select'>
+         {options.map((x)=> {
+            return (
+               <option value={x.id}>{x.label}</option>
```

Comment:
`key` is missing. Keys are a must, read here why it's important https://reactjs.org/docs/lists-and-keys.html
If you check console you'll see a bunch of warnings about it, always check it.